sql/core/src/test/resources/sql-tests/inputs/udf/postgreSQL/udf-aggregates_part3.sql

```diff
@@ -247,7 +247,8 @@ select udf(max(min(unique1))) from tenk1;
 -- from (values ('a', 'b')) AS v(foo,bar);
 
 -- outer reference in FILTER (PostgreSQL extension)
-select udf((select udf(count(*))
+-- modified to remove internal udf
+select udf((select count(*)
```

Comment:
Hm, @LucaCanali I think we should ideally make the existing case working instead of avoiding it in the test. Can we try to make the matrix lazy val, and assign it to the local variable, and use it in `rdd.mapPartitions` block at `doExecite`?